ui/src/hooks/useKubernetes.ts

```diff
@@ -496,8 +496,8 @@ export function createConnect(t: any) {
         handleTreeMessage(ws, event);
         handleTerminalMessage(ws, event, createSentry, t);
       },
-      onError: () => handleInterrupt('error'),
-      onDisconnected: () => handleInterrupt('disconnected'),
+      onError: () => handleInterrupt('error', t),
+      onDisconnected: () => handleInterrupt('disconnected', t),
     });
 
     return ws.value;
```

Comment:
The above code appears to be incomplete. Please share the complete codebase for me to review and provide more specific feedback about potential differences, issues, optimizations or improvements.